app/src/main/java/nu/yona/app/api/model/YonaUser.java

```diff
@@ -32,6 +32,32 @@
 	@SerializedName("mobileNumber")
 	@Expose
 	private String mobileNumber;
+	@SerializedName("nickname")
+	@Expose
+	private String nickname;
+	@SerializedName("_embedded")
+	@Expose
+	private EmbeddedDevicesGoals embedded;
```

Comment:
Logically yes. It is not an issue because buddy data is not used from the persisted data before the first user update call returns unlike earlier properties (eg: postOpenAppEvent, postDeviceAppActivity).

Reply:
Do it, even though it is not going to fail today. A small change in the logic might cause a failure in a later stage. The rule should be to update the version number when the data structure changes.